specutils/tests/test_fitting.py

```diff
@@ -42,7 +46,41 @@ def double_peak():
     return x, y_double
 
 
-def test_single_peak_fit():
+def double_peak_absorption_and_emission():
+    np.random.seed(42)
+    g1 = models.Gaussian1D(1, 4.6, 0.2)
+    g2 = models.Gaussian1D(2.5, 5.5, 0.1)
+    g3 = models.Gaussian1D(-1.7, 8.2, 0.1)
+    x = np.linspace(0, 10, 200)
+    y_double = g1(x) + g2(x) + g3(x) + np.random.normal(0., 0.2, x.shape)
+    return x, y_double
+
+
+def test_find_lines_derivative():
+
+    # Create the spectrum to fit
+    x_double, y_double = double_peak_absorption_and_emission()
+    spectrum = Spectrum1D(flux=y_double*u.Jy, spectral_axis=x_double*u.um)
+
+    # Threshold method
+    lines = find_lines_threshold(spectrum, sigma=0.75)
```

Comment:
the test is called derivative but this is for threshold?  I'd say you should either split these up or re-name the function.

Reply:
I split them up